templates/eap-xp1-openjdk11-image-stream.json

```diff
@@ -0,0 +1,135 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "jboss-eap-xp1-openjdk11-openshift",
+        "annotations": {
+            "description": "ImageStream definition for JBoss Enterprise Application Platform Expansion Pack with OpenJDK 11",
```

Comment:
official name is "JBoss Enterprise Application Platform expansion pack" (minor case)